src/main/resources/application.yaml

```diff
@@ -16,12 +16,12 @@ spring:
 
   data:
     mongodb:
-      authentication-database: ${mongo.database.name:rental-db}
+      authentication-database: ${mongo.database.name:rental}
```

Comment:
let's simply use rental as db name, no need for configuration through mongo.database.name